controllers/user.js

```diff
@@ -20,6 +20,14 @@ const {
 
 const FXA_MONITOR_SCOPE = "https://identity.mozilla.com/apps/monitor";
 
+//DATA REMOVAL SPECIFIC
+const { FormUtils } = require("../form-utils");
+const fetch = require("node-fetch");
```

Comment:
This repo is already using "got" for Node requests: https://www.npmjs.com/package/got

I'm more used to node-fetch as well, but assuming "got" fits your needs, maybe we should stick to a single Node request library?  I think the usage is similar: 
`response = await got(url, reqOptions);`

Reply:
@toufali  - How'd you feel about adding this as a separate ticket? I tried a simple find and replace which didn't work, so I'll need to make sure I understand the syntax well and how it handles the various options we need in fetching data. 

Since this is the core functionality of the service, I'm hesitant to do this as part of the larger merge because I'm not sure what I'll be messing up in the process.

Reply:
All good!  This is just a nice-to-have; not imperative.